src/lib/utils/env.js

```diff
@@ -0,0 +1,12 @@
+import config from '%src/config'
+import constants from '%src/constants/env'
+
+const isDevelopmentEnv = () => config.NODE_ENV === constants.ENV_NAMES.DEVELOPMENT
+const isTestEnv = () => config.NODE_ENV === constants.ENV_NAMES.TEST
+const isProductionEnv = () => config.NODE_ENV === constants.ENV_NAMES.PRODUCTION
+
+export default {
```

Comment:
In other util files we use `export const isDevelopmentEnv = ...` etc etc

This pattern eliminates the annoying `prefer export default` problem we face when a utility method only has a single export, but it is a bit more verbose.  That said, it is also a bit more explicit about what is exported.

We should probably pick one and stick with elsewhere in the project.  cc @reefdog -- what do you think?

Reply:
Changed in latest commit.